src/v2/ion/responseTransformer.js

```diff
@@ -229,13 +229,12 @@ const convert = (settings, idx = {}, lastResult = null) => {
   );
 
   injectIdPConfigButtonToRemediation(settings, result);
+  modifyFormNameForIdPAuthenticator(result);
```

Comment:
@haishengwu-okta - When we made this change with #1780, we didn't account for the IdP Authenticator having a single remediation response of `redirect-idp`. As a result, we'd auto-redirect to the IdP when we're receive a response [like this](https://github.com/okta/okta-signin-widget/blob/998bcba0855c0ed32f27a79710745c12ae6215f5/playground/mocks/data/idp/idx/authenticator-verification-idp-single-remediation.json).

Invoking the form-name change here ensures the `convertRedirectIdPToSuccessRedirectIffOneIdp` doesn't accidentally trigger the redirect.